File: Assets/Scripts/MenuTextControl.js
#pragma strict

function Start () {
    GetComponent.<Renderer>().material.color = Color.gray;
}

function OnMouseEnter () {
    GetComponent.<Renderer>().material.color = Color.red;
}

function OnMouseExit () {
    GetComponent.<Renderer>().material.color = Color.gray;
}

function OnMouseUp () {
    //load the game
    Application.LoadLevel(1);
}

File: Assets/Scripts/MenuTextControl.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.SceneManagement;

public class MenuTextControl : MonoBehaviour {
    void Start () {
        GetComponent<Renderer>().material.color = Color.gray;
    }

    void OnMouseEnter () {
        GetComponent<Renderer>().material.color = Color.red;
    }

    void OnMouseExit () {
        GetComponent<Renderer>().material.color = Color.gray;
    }

    void OnMouseUp () {
        //load the game
        SceneManager.LoadScene("FutureCop");
    }
}
